modules/cluster-logging-deploy-cli.adoc

```diff
@@ -58,7 +58,7 @@ spec:
   targetNamespaces:
   - openshift-logging <1>
 ----
-<1> You must specify the `openshift-logging` namespace for logging versions 5.7 and older. For logging 5.8 and later versions, you can use any namespace.
+<1> You must specify the `openshift-logging` namespace for logging versions 5.7 and older. However, continuing to use this namespace prevents you from using the multi log forwarder feature. For logging 5.8 and later versions, you can use any namespace, as it enables the multi log forwarder feature.
```

Comment:
Starting in RHOL 5.8, the operator can be installed in the `openshift-logging` namespace, in any other namespace/s not being  `openshift-logging`  that should be listed in the "targetNamespaces" or in **all** the namespaces. 

By default, in RHOL 5.8 or posterior versions, if it's installed from the OCP Console, it's installed in "all" namespaces.

Reply:
Revised sentence, "For logging versions 5.7 and older, the `openshift-logging` namespace must be specified. For logging versions 5.8 and later, the operator can be installed in the `openshift-logging` namespace, any other namespace, or in `All Namespaces`."